helm/flink-operator/values.yaml

```diff
@@ -23,9 +23,9 @@
 # watchNamespaces: ["flink"]
 
 image:
-  repository: flink-operator
+  repository: ghcr.io/apache/flink-operator
   pullPolicy: IfNotPresent
-  tag: latest
+  tag: main
```

Comment:
I may prefer use latest in the CD workflow which is the default manner in the Docker world. Could you explain a bit why do we use `main` instead of `latest` in the CD workflow? IIRC `docker pull <image>` is default to `latest`.

Reply:
Given that this is the snapshot docker image of latest main branch, I also think `latest` tag might be better here.